### PR TITLE
Get rid of the vars all around DelA code

### PR DESCRIPTION
**Pull request by jbartece**
Sometimes it can make sense to use var, but this usage makes the whole codebase very unclean and hard to read and understand, if you are not the one, who wrote the code

### CODE REVIEW

File: src/main/java/org/jboss/pnc/deliverablesanalyzer/rest/AnalyzeResource.java [hunk 1]
```diff
@@ -18,10 +18,7 @@
 import java.io.IOException;
 import java.net.URI;
 import java.time.Duration;
-import java.util.concurrent.CompletionStage;
-import java.util.concurrent.ExecutionException;
-import java.util.concurrent.TimeUnit;
-import java.util.concurrent.TimeoutException;
+import java.util.concurrent.*;
```
Comment (janinko) [inline src/main/java/org/jboss/pnc/deliverablesanalyzer/rest/AnalyzeResource.java]: don't use star import

File: src/main/java/org/jboss/pnc/deliverablesanalyzer/rest/ErrorMessage.java [hunk 2]
```diff
@@ -53,9 +53,9 @@ public ErrorMessage(Exception exception) {
         Response.Status status = Response.Status.INTERNAL_SERVER_ERROR;
 
         if (exception instanceof WebApplicationException) {
-            var e = (WebApplicationException) exception;
+            WebApplicationException e = (WebApplicationException) exception;
```
Comment (janinko) [inline src/main/java/org/jboss/pnc/deliverablesanalyzer/rest/ErrorMessage.java:56]: don't need to cast the value

Comment (jbartece) [reply src/main/java/org/jboss/pnc/deliverablesanalyzer/rest/ErrorMessage.java:56]: If I don't cast it, IDE screams and shows an error

File: src/main/java/org/jboss/pnc/deliverablesanalyzer/rest/AnalyzeResource.java (hunk 1 above)
Comment (jbartece) [reply src/main/java/org/jboss/pnc/deliverablesanalyzer/rest/AnalyzeResource.java]: Ah, bad IDE setting... I'll fix it

File: src/main/java/org/jboss/pnc/deliverablesanalyzer/rest/ErrorMessage.java (hunk 2 above)
Comment (janinko) [reply src/main/java/org/jboss/pnc/deliverablesanalyzer/rest/ErrorMessage.java:56]: yes, sorry, it's an `if instanceof` not `catch`

**Pull request merged**